### ISSUE TITLE
Permit the removal of nicks and clipped indications in split messages

### ISSUE THREAD

**Issue by dollhaus** [42wim/matterbridge#2087]
This is primarily a request for IRC, but could potentially work on other services.

At present, a split message in IRC looks like this:

bridge | <username> : foo <clipped message>
bridge | <username> : bar <clipped message>
bridge | <username> : baz

When a message covers multiple lines, this can be difficult to read. I'd like to have the option of the following:

bridge | <username> : foo
bridge | bar
bridge | baz

In the second example, both the handle of the speaker and the <clipped message> indicator are removed, relying on the flow of conversation instead of using delineators. 

Here is an example of where this could be helpful: 

![Capture](https://github.com/42wim/matterbridge/assets/33754407/fa0f584b-5662-49a3-8284-4da7ef90b4d2)

As you can see, this discussion is somewhat hard to follow because the flow of text is disturbed by the presence of a handle and delineator in the middle of it. Removing these would make the text much easier to read.

The only potential issue I see is the possibility of messages being interrupted by another speaker - but in smaller channels this is rare, and I don't see it being a problem.

**Comment by bkil** [comment]
I do not support this. This request could interfere when using multiple relay bots, or a script on a bouncer or client that would reformat received messages from relay bots to attribute them to their original senders. If a message line is stripped from the sender information, the forwarded message will thus be misattributed. Some examples to help you understand this better:

* https://github.com/42wim/matterbridge/issues/854 ExtractNicks
* https://www.emacswiki.org/emacs/rcircForDiscordBridge
* https://github.com/SrainApp/srain/blob/master/docs/faq.rst#what-is-relay-message-transform
